app/services/status_handler.rb

```diff
@@ -12,7 +12,7 @@ def initialize(audit_results:, complete_moab:)
     @complete_moab = complete_moab
   end
 
-  def update_status(new_status)
+  def update_complete_moab_status(new_status)
```

Comment:
Also did some renaming of StatusHandler methods to make them clearer.

Reply:
super!  But is this better named 
- update_moab_catalog_status
- update_catalog_moab_status
- update_db_moab_status 

or something?

Reply:
🤷🏻‍♂️

Reply:
So it is updating the complete moab, which is the current name. If in the future the complete moab get renamed, then this method should get renamed accordingly.